rca/people/utils.py

```diff
@@ -0,0 +1,46 @@
+from django.apps import apps
```

Comment:
The functions here are based off the existing work in https://github.com/torchbox/rca-wagtail-2019/blob/84ad028150dbc961fcf8e240e982ab30998861b2/rca/people/models.py#L216-L235

Rather than duplicating them entirely in the model, I've moved them here; with an aim to making them one function/class